preliz/internal/plot_helper.py

```diff
@@ -606,7 +606,11 @@ def check_inside_notebook(need_widget=False):
             )
         if need_widget:
             shell_name = shell.__class__.__name__
-            if shell_name == "ZMQInteractiveShell" and "nbagg" not in get_backend():
+            if shell_name == "ZMQInteractiveShell" and get_backend() not in [
+                "ipympl",
+                "widget",
+                "nbagg",
+            ]:
```

Comment:
```suggestion
            if shell_name == "ZMQInteractiveShell" and get_backend().split('_')[-1].lower() not in [
                "ipympl",
                "widget",
                "nbagg",
            ]:
```
It should be something like this

Reply:
Thank!!
Done :)